src/Momento.Sdk/Responses/CacheSetLengthResponse.cs

```diff
@@ -0,0 +1,105 @@
+using Momento.Protos.CacheClient;
+using Momento.Sdk.Exceptions;
+
+namespace Momento.Sdk.Responses;
+
+/// <summary>
+/// Parent response type for a cache set length request. The
+/// response object is resolved to a type-safe object of one of
+/// the following subtypes:
+/// <list type="bullet">
+/// <item><description>CacheSetLengthResponse.Hit</description></item>
+/// <item><description>CacheSetLengthResponse.Miss</description></item>
+/// <item><description>CacheSetLengthResponse.Error</description></item>
+/// </list>
+/// Pattern matching can be used to operate on the appropriate subtype.
+/// For example:
+/// <code>
+/// if (response is CacheSetLengthResponse.Hit hitResponse)
+/// {
+///     return hitResponse.Length;
+/// }
+/// else if (response is CacheSetLengthResponse.Miss missResponse)
+/// {
+///     // handle missResponse as appropriate
+/// }
+/// else if (response is CacheSetLengthResponse.Error errorResponse)
+/// {
+///     // handle error as appropriate
+/// }
+/// else
+/// {
+///     // handle unexpected response
+/// }
+/// </code>
+/// </summary>
+public abstract class CacheSetLengthResponse
+{
+    /// <include file="../docs.xml" path='docs/class[@name="Hit"]/description/*' />
+    public class Hit : CacheSetLengthResponse
+    {
+        /// <summary>
+        /// The length of the set. If the set is missing or empty, the result is zero.
+        /// </summary>
```

Comment:
The result is a miss here, right? Looking at the code in the data client we return a `Miss` if not found.